README.md

```diff
@@ -209,12 +193,7 @@ Don't forget to give the project a star! Thanks again!
 * [Img Shields](https://shields.io)
 * [React Icons](https://react-icons.github.io/react-icons/search)
 
-<p align="right">(<a href="#top">back to top</a>)</p>
-
-
 <!-- LICENSE -->
 ## License
 
 Distributed under the **A-GPL** License. See [`LICENSE`](./LICENSE) for more information.
```

Comment:
## Optional

Better to write the license as AGPL. No where you can find A-GPL because it is AGPL.

Maybe check https://en.wikipedia.org/wiki/Affero_General_Public_License. Even in the last part of the license file, they have written AGPL. I think on such legal things, it is better to follow the norm.

This is optional but I strongly recommend you doing it.